installation/proc-install-rhdh-operator.adoc

```diff
@@ -272,10 +272,12 @@ The internal OpenShift cluster image registry cannot be used as a target mirror
 * The `htpasswd` command is available. For Red Hat Enterprise Linux, the `htpasswd` command is available by installing the `httpd-tools` package.
 
 .Procedure
-. Download and run the mirroring script to install a custom Operator catalog and mirror the related images: `prepare-restricted-environment.sh` (link:https://github.com/janus-idp/operator/blob/1.1.x/.rhdh/scripts/prepare-restricted-environment.sh[source]).
+. Download and run the mirroring script to install a custom Operator catalog and mirror the related images: `prepare-restricted-environment.sh` (link:https://github.com/janus-idp/operator/blob/{product-version}.x/.rhdh/scripts/prepare-restricted-environment.sh[source]).
 +
 [source,yaml]
 ----
+curl -sSLO https://raw.githubusercontent.com/janus-idp/operator/{product-version}.x/.rhdh/scripts/prepare-restricted-environment.sh
```

Comment:
I'm hoping this works; not sure if adoc vars are expanded within source blocks?

Reply:
This will work.